<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterDetailDialog</class>
 <widget class="QDialog" name="CharacterDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>1060</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Character Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="hpBar">
     <property name="format">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="mpBar">
     <property name="format">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cooldownQValuesLayout">
     <item>
      <widget class="QListWidget" name="skillCooldownList">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="qValuesTable"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="stateMachineLabel">
     <property name="text">
      <string>State Machine</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
